homeassistant/components/ecoforest/strings.json

```
@@ -17,5 +17,40 @@
     "abort": {
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "cpu_temperature": {
+        "name": "CPU Temperature"
```

```suggestion
        "name": "CPU temperature"
```
Please also change this at the rest. Only Title case, unless its a brand name or something